kit/clipboard-copy/index.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+
+import copy from 'clipboard-copy';
+
+export {copy as copyToClipboard};
```

Comment:
John -- why do we need this?  Would it be possible to do this with the 'clipboard' package that we already have.....

Reply:
I looked at clipboard.js and didn't see any way to trigger the copy to clipboard programmatically. It looked like it always needed to be attached to an element in order to function, which would be a little tricky with what ag-Grid provides us, and would generally require a lot more wiring to function if it is even possible. Unless I am missing something it definitely would not be as simple as the one-liner that the clipboard-copy package provides.